config/default/kustomization.yaml

```diff
@@ -24,3 +24,4 @@ resources:
 - ../crd
 - ../rbac
```

Comment:
@chr15p lets keep the config changes for later PRs. We would like to get all the controllers/pkg code ready, and once it is done we will add the configuration and switch to the new code. This way we are keeping PRs smaller and also let the operator be deploy-able (with the old version) at the same time